Header header
string[] namespaces
uint32[] face_scores
uint32 num_robot
bool is_valid
